frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -53,6 +53,10 @@
     return a.user.name.localeCompare(b.user.name);
   });
 
+  $: lexEntryCount = data.lexEntryCount;
+  // Doesn't work: we get the following error.
+  // Cannot call `fetch` eagerly during server side rendering with relative URL (/api/countLexEntries/sena-3) — put your `fetch` calls inside `onMount` or a `load` function instead
```

Comment:
what doesn't work? Is this still relevant?

Reply:
No longer relevant, I missed deleting this comment when I fixed the issue. I'll delete the comment.